csp/storage/storage.go

```diff
@@ -0,0 +1,53 @@
+package storage
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/vocdoni/saas-backend/internal"
+)
+
+// Storage interface implements the storage layer for the smshandler
+type Storage interface {
```

Comment:
Do we really need this interface?

Reply:
I don't know, right now i use it to guide the development.